core/src/com/fallenflame/game/FogController.java

```diff
@@ -72,9 +72,10 @@ public void updateFogAndDraw(GameCanvas canvas, Vector2 scale, float delta) {
                 }
                 //To prevent drawing on tiles with the player or a wall as well as if its within the light radius
                 if (levelModel.hasWall(x, y)) continue;
+                //0.5 accounts for the aligning of the light to show the player's face over the feat.
```

Comment:
What does "show the player's face over the feat" mean?

Reply:
It accounts for the fix Ronnie did way back w/ the light radius not showing the players face. If you look at an old branch, you'll notice the black circle is offset. I didn't make it a constant cause it's used once.

Reply:
Oh do you mean "feet"?

Reply:
...yes